### ISSUE TITLE
default channels per page & sort options

### ISSUE THREAD

**Issue by seth586** [Ride-The-Lightning/RTL#296]
Every time I use RTL I am clicking "channels per page: 100" and then clicking "peer" to sort alphabetically.

It would be nice to reduce this workflow for my routing node maintenance routine by giving us options to set defaults in `RTL-Config.json` such as (example):
```
"channelsperpage": "100",
"peersperpage": "100",
"routingSort": "timestamp",
```

etc....

**Comment by ShahanaFarooqui** [comment]
@saubyk As discussed earlier, we were not in favor of adding these many configurations to RTL. But, it looks like a very popular feature request.

Do you want to reconsider your decision? If yes, we need to discuss the UX/UI for the feature.

**Comment by saubyk** [comment]
@ShahanaFarooqui we can consider creating a small database for config options.
It can help us scale other capabilities as well on the application.

**Comment by mjlamb** [comment]
Also sorting by the peer in the list is not valid, CASE letters are taking precedent in the alias name sort and is not sorted Alpha correctly. i.e. an alias with uppercase B is sorted at top and lower case b is further below in list.

**Comment by mjlamb** [comment]
The latest release has resolved the "peer" column sorting, but sats column sorting is still broken.... does not sort sats correctly.

1,043,390
1,081,790
114,665
119,643

etc

**Comment by ShahanaFarooqui** [comment]
@mjlamb Though I was unable to reproduce 'Sats' sorting issue on my local, I hope that the latest commit at [0.8.4 branch] (https://github.com/Ride-The-Lightning/RTL/tree/Release-0.8.4) will fix it from v.0.8.4.

**Comment by viaj3ro** [comment]
would you consider adding `all` so it's possible to show (and search) all channels/peers in one go, not just the first 100?

**Comment by mjlamb** [comment]
Should default to showing 100 or All, which is the smallest.

**Comment by viaj3ro** [comment]
yeah, but once you are above 100 channels, there is now way to show them all on one page at the moment. So adding the option to show all, no matter what, would do the trick.

**Comment by mjlamb** [comment]
Only able to show 100? That is a strange constraint.

**Comment by sendbitcoin** [comment]
I would also like an option to list all channels at once instead of just max 100.

also PLEASE add the "channels per page" dropdown to the TOP of the page as well. currently I have to scroll all the way down, change it to 100, then scroll all the way back up as it puts you at the bottom of the page after setting it to 100. Its very annoying. Adding the dropdown to the top of the page as well would massively improve the UX for when you are looking at channels and need to sort them in various ways.

**Comment by ShahanaFarooqui** [comment]
Closing it with PR #1127. Released with v0.13.2.
